config/pipeline.yaml

```diff
@@ -170,6 +170,21 @@ runtimes:
         token: kernelci-api-token-lava-staging
         url: https://staging.kernelci.org:9100
 
+  lava-baylibre:
+    lab_type: lava
+    url: 'https://lava.baylibre.com/'
+    priority_min: 50
+    priority_max: 70
```

Comment:
IIRC those aren't handled in the new system, you can drop those lines.

Reply:
Yes, that's true. I'll create a new issue for that.

Reply:
Created https://github.com/kernelci/kernelci-pipeline/issues/513